gsoft-inc/sg-orbit
# Switched chromatic infra to build and upload

Issue: https://github.com/gsoft-inc/sg-orbit/issues/59

## What I did

- Updated the scripts to use the build & upload infra of chromatic instead of the tunneling infra
- Also added scripts to serve a build version of storybook, it's usefull to debug.

## How to test

- Is this testable with Jest or Chromatic screenshots? Kind of
- Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.
